src/ray/core_worker/transport/actor_task_submitter.cc

```diff
@@ -579,7 +579,7 @@ void ActorTaskSubmitter::PushActorTask(ClientQueue &queue,
   request->mutable_task_spec()->CopyFrom(task_spec.GetMessage());
 
   request->set_intended_worker_id(queue.worker_id);
-  request->set_sequence_number(queue.actor_submit_queue->GetSequenceNumber(task_spec));
```

Comment:
Another follow-up is renaming ActorCounter to probably just SequenceNumber.

Reply:
https://github.com/ray-project/ray/pull/52972